subgraphs/uniswap-forks/protocols/uniswap-v2/src/common/constants.ts

```diff
@@ -0,0 +1,12 @@
+////////////////////
+//////Versions//////
+////////////////////
+
+import { log } from "@graphprotocol/graph-ts";
+
+export const PROTOCOL_SUBGRAPH_VERSION = "1.1.2";
+export const PROTOCOL_METHODOLOGY_VERSION = "1.0.0";
+export const PROTOCOL_NAME = "Uniswap V2";
+export const PROTOCOL_SLUG = "uniswap-v2";
+
+log.warning("HELLO52", [])
```

Comment:
what's this for?

Reply:
Just used to trigger the deployments in testing. When I removed a commit, it was a change I did not remove again.

Reply:
I removed it.